### PR TITLE
Feature/us 50 put operation to update users

### PR DESCRIPTION
**Pull request by Gra2411**

### CODE REVIEW

File: pom.xml [hunk 1]
```diff
@@ -71,12 +71,12 @@
 			<groupId>org.springframework.boot</groupId>
 			<artifactId>spring-boot-starter-test</artifactId>
 			<scope>test</scope>
-			<exclusions>
-				<exclusion>
-					<groupId>org.junit.vintage</groupId>
-					<artifactId>junit-vintage-engine</artifactId>
-				</exclusion>
-			</exclusions>
+<!--			<exclusions>-->
```
Comment (ivanflores15) [inline pom.xml]: If you will not use this exclusion is better remove it

File: src/main/java/com/agilethought/internship/sso/controller/ControllerApplication.java [hunk 2]
```diff
@@ -0,0 +1,52 @@
...
+import java.util.List;
+
+@RestController
+@Slf4j
+public class ControllerApplication {
+
+	@Autowired
+	ServiceApplication serviceApplication;
+
+	@PostMapping(value = "/api/v1/user", produces = "application/json")
```
Comment (ivanflores15) [inline src/main/java/com/agilethought/internship/sso/controller/ControllerApplication.java]: As far I remember we include @RequestMapping tag to define base path, is that rigth?

File: src/main/java/com/agilethought/internship/sso/controller/GlobalExceptionHandler.java [hunk 3]
```diff
@@ -1,44 +0,0 @@
-package com.agilethought.internship.sso.controller;
```
Comment (ivanflores15) [inline src/main/java/com/agilethought/internship/sso/controller/GlobalExceptionHandler.java]: Don't remove this class, it belongs to previuos behaviors, you need to adapt your code to this handler

File: src/main/java/com/agilethought/internship/sso/exception/BadRequestException.java [hunk 4]
```diff
@@ -3,25 +3,11 @@
 import org.springframework.http.HttpStatus;
 import org.springframework.web.bind.annotation.ResponseStatus;
 
-import lombok.Getter;
```
Comment (ivanflores15) [inline src/main/java/com/agilethought/internship/sso/exception/BadRequestException.java:6]: Don't change previous behaviors for handler exception

File: src/main/java/com/agilethought/internship/sso/exception/HttpMessageNotReadableException.java [hunk 5]
```diff
@@ -1,27 +0,0 @@
-/*package com.agilethought.internship.sso.exception;
```
Comment (ivanflores15) [inline src/main/java/com/agilethought/internship/sso/exception/HttpMessageNotReadableException.java:1]: Don't remove

File: src/main/java/com/agilethougth/intership/sso/errorhandling/HttpExceptionMessage.java [hunk 6]
```diff
@@ -1,17 +0,0 @@
-package com.agilethougth.intership.sso.errorhandling;
```
Comment (ivanflores15) [inline src/main/java/com/agilethougth/intership/sso/errorhandling/HttpExceptionMessage.java]: don't remove

File: src/main/java/com/agilethougth/intership/sso/errorhandling/PathErrorMessage.java [hunk 7]
```diff
@@ -1,6 +0,0 @@
-package com.agilethougth.intership.sso.errorhandling;
```
Comment (ivanflores15) [inline src/main/java/com/agilethougth/intership/sso/errorhandling/PathErrorMessage.java]: Don't remove

File: src/main/resources/log4j2.properties [hunk 8]
```diff
@@ -1,29 +1,20 @@
+status=error
```
Comment (ivanflores15) [inline src/main/resources/log4j2.properties:1]: Don't change previous behavior

**Review by ivanflores15 — CHANGES_REQUESTED** [review]
Generally, don't change previuos behavior for error Handling or log4j

**Comment by ivanflores15** [comment]
please rebase to current develop reference

**Review by luisgalindoluna — COMMENTED** [review]
I noticed your approach for this was to delete some classes and modify the behavior in the Implementation so it wouldn't use them anymore and then paste our classes in the ServiceImpl.
I don't doubt that it works, but is that how we wanted to do it? I was under the impression we'd migrate our own classes rather than modify the existing ones.

**Pull request closed without merging**